.github/workflows/deploy.yml

```
@@ -0,0 +1,252 @@
+          ssh ${{ secrets.EC2_USER }}@${{ secrets.EC2_HOST }} '
+            echo "🚀 Starting deployment..."
+            cd /tmp
+            
+            # Pull new image
+            echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+            docker pull ${{ env.IMAGE }}
+            
+            # Update docker-compose to use new image
+            export NEW_IMAGE="${{ env.IMAGE }}"
+            sed -i "s|image:.*know.*|image: $NEW_IMAGE|" docker-compose.yml
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)